core-profile/rest-tck/pom.xml

```diff
@@ -172,6 +161,7 @@
                                     <include name="**/json/bind/api/main/*.jar"/>
                                     <include name="**/inject/api/main/*.jar"/>
                                     <include name="**/interceptor/api/main/*.jar"/>
+                                    <include name="**/mail/api/main/*.jar"/>
```

Comment:
Why?

Reply:
There is a new test for a new multipart test. The current RESTEasy implementation uses Apache Mim4j, which requires Jakarta Mail.